### ISSUE TITLE
Component Accessibility Status: EAC default state baseline coverage

### ISSUE THREAD

**Issue by elycheea** [carbon-design-system/ibm-products#4626]
### Acceptance criteria

- There are no violations when running the automated [Equal Access Checker](https://www.ibm.com/able/toolkit/tools) on default component state.
- Ensure default state are include all components/patterns.
- ID should match Story name
- Manually add e2e for accessibility check in default stories

```
    await expect(page).toHaveNoACViolations('Cascade @avt-default-state');
```

REF #4345 #4631

```[tasklist]
### Tasks
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4660
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4628
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4661
- [x] Cascade
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4662
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4663
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4664
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4665
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4666
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4667
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4668
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4669
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4670
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4671
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4672
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4673
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5001
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5002
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5003
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5004
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5005
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5006
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5007
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5008
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5009
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5010
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5059
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5060
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5061
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5062
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5063
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5064
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5065
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5066
```

## Internal

```[tasklist]
### Tasks
- [ ] ~ActionSet~
- [ ] ~BreadcrumbWithOverflow~
- [ ] ~ButtonSetWithOverflow~
- [ ] ~Carousel~
- [ ] ~DecoratorBase~
- [ ] ~FilterPanelLabel~
- [ ] ~FilterSummary~
- [ ] ~SimpleHeader~
- [ ] ~SteppedAnimatedMedia~
- [ ] ~TearsheetShell~
- [ ] ~TooltipTrigger~
```

## (Soon-to-be) deprecated

```[tasklist]
### Tasks
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5069
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5070
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5071
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5072
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5073
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5074
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5075
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5076
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5077
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5078
```

## Lower priority

Lower usage should still include AVT coverage since these are formally released.
Unstable/Canary components do not require AVT yet but should be added if/when they are being released as stable.

```[tasklist]
### Lower usage
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5088
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5089
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5090
```

## Unstable

```[tasklist]
### Tasks
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5205
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5206
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5207
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5208
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5209
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5210
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5211
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5134
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5135
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5213
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5136
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5212
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5214
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5215
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5216
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5217
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5218
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5219
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5220
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5221
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5223
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5224
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5932
```
